portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,37 @@
   width: 650px;
 }
 
-#greeting-container {
+#opinion-container {
   margin-top: 20px;
 }
+
+a:link, a:visited {
+  color: black;
+}
+
+body {
+  background-color: #ecc377;
+  color: black;
+  font-family: sans-serif;
+}
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  height: 50%;
```

Comment:
Another minor style thing: Google's CSS style guide recommends alphabetizing CSS rules: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

I actually don't like this rule, so I never follow it in my personal projects. I have to follow it at work though. I'll leave it up to you whether you want to follow it or not- whatever you think is easier to read is okay with me!